query/sources/impl/src/test/groovy/org/codice/ddf/admin/sources/SourceTestCommons.groovy

```diff
@@ -77,28 +77,39 @@ class SourceTestCommons {
     static TEST_SOURCENAME = "testSourceName"
 
     static discoverByAddressActionArgs = [
-        (ADDRESS): [
-            (HOST)       : [
-                (PORT)    : 8993,
-                (HOSTNAME): "localhost"
-            ]
-        ],
-        (CREDENTIALS): [
-                (USERNAME): TEST_USERNAME,
-                (PASSWORD): TEST_PASSWORD
-        ]
+        (ADDRESS) : new AddressField().hostname('localhost').port(8993).getValue(),
+        (CREDENTIALS) : new CredentialsField().username(TEST_USERNAME).password(TEST_PASSWORD).getValue()
     ]
 
     static discoverByUrlActionArgs = [
-        (ADDRESS) : [
-            (URL_NAME)   : "http://localhost:8993/sevices/csw",
-        ],
-        (CREDENTIALS): [
-                (USERNAME): TEST_USERNAME,
-                (PASSWORD): TEST_PASSWORD
-        ]
+        (ADDRESS) : new AddressField().url("http://localhost:8993/sevices/csw").getValue(),
+        (CREDENTIALS) : new CredentialsField().username(TEST_USERNAME).password(TEST_PASSWORD).getValue()
     ]
 
+    static saveConfigActionArgs = [
```

Comment:
I would avoid using static objects. I would make a method that returns you a new one instead, then you won't have to refresh or run into cross test contamination.